charts/pega/charts/installer/templates/pega-installer-role.yaml

```diff
@@ -5,7 +5,13 @@ metadata:
   name: {{ template "installerJobReaderRole" }}
   namespace: {{ .Release.Namespace }}
 rules:
-- apiGroups: ["", "batch", "extensions", "apps"]
```

Comment:
I didn't the reason for removing core API group.

Reply:
I restricted the rules, as it was not working on the cluster I'm deploying on. But with these restricted rights, the installer deployment is working fine.